docs/reference/announcements.md

```diff
@@ -4,6 +4,24 @@ title: "Announcements"
 description: "Important announcements including deprecation & removal notices"
 ---
 
+
+## Camunda 8.5
+
+
+### Support for Amazon OpenSearch
+As of the 8.5 release, Optimize is now also compatible with [Amazon OpenSearch](https://aws.amazon.com/de/opensearch-service/) 2.5.x. Note that using Amazon OpenSearch requires [setting up a new Camunda installation](/self-managed/platform-deployment/overview.md). A migration from previous versions or Elasticsearch environments is currently not supported.
+
```

Comment:
Probably out of scope here, but the page you're linking to makes reference that Operate, Tasklist and Optimize require ElasticSearch. This should be updated to ElasticSearch/OpenSearch

Reply:
Are we still looking to clean this up?

Reply:
@toco-cam Are you still looking for someone to update here? Not sure if Amara got to it before she headed out.

Reply:
@christinaausley if @akeller did not do yet, yes.

Reply:
Merging this PR in, but let me know if I can assist here with a follow up PR @akeller 👍

Reply:
This fell off my radar. I will create an issue and PR this week.

Reply:
Ah! This is no longer relevant. This page was reworked and the dependencies were moved to the Supported Environments page. That's why I didn't do it 🙂 Resolving this conversation